src/commands/token/exchangeTokenCommand.ts

```diff
@@ -0,0 +1,53 @@
+import { ExchangeTokenInterface } from "../../interfaces/token";
+
+export default class ExchangeTokenCommand {
```

Comment:
same here, let's just create an interface and pass an object around, like authorizationCommand